docs/secrets.md

```diff
@@ -243,6 +252,54 @@ parameters:
         VAULT_CLIENT_KEY: /path/to/key
         VAULT_CLIENT_CERT: /path/to/cert
 ```
+
+### Vaulttransit Secret Backend - Addons
+
+Considering a key-value pair like `my_key`:`my_secret` in the path `secret/foo/bar` in a transit secret engine on the vault server, to use this as a secret use:
+
+```shell
+$ echo "any.value:whatever-you_may*like"  | kapitan refs --write vaulttransit:my_target/to/secret_inside_kapitan -t <target_name> -f -
+```
+
+Parameters in the secret file are collected from the inventory of the target we gave from CLI `-t <target_name>`. If target isn't provided then kapitan will identify the variables from the environment when revealing secret.
+
+Environment variables that can be defined in kapitan inventory are `VAULT_ADDR`, `VAULT_NAMESPACE`, `VAULT_SKIP_VERIFY`, `VAULT_CLIENT_CERT`, `VAULT_CLIENT_KEY`, `VAULT_CAPATH` & `VAULT_CACERT`.
+Extra parameters that can be defined in inventory are:
+* `auth`: specify which authentication method to use like `token`,`userpass`,`ldap`,`github` & `approle`
+* `mount`: specify the mount point of key's path. e.g if path=`my_mount` (default `transit`)
+* `crypto_key`: Name of the `encryption key` defined in vault
+* `always_latest`: Always rewrap ciphertext to latest rotated crypto_key version
+Environment variables cannot be defined in inventory are `VAULT_TOKEN`,`VAULT_USERNAME`,`VAULT_PASSWORD`,`VAULT_ROLE_ID`,` VAULT_SECRET_ID`.
+
+```yaml
+parameters:
+  kapitan:
+    vars:
+      target: my_target
+      namespace: my_namespace
+    secrets:
+      vaulttransit:
+        VAULT_ADDR: http://vault.example.com:8200
+        VAULT_TOKEN: s.i53a1DL83REM61UxlJKLdQDY
```

Comment:
and here

Reply:
see also: https://github.com/kapicorp/kapitan/pull/826#discussion_r829709083
We should hint this in the docs like [this](https://github.com/kapicorp/kapitan/pull/826#discussion_r829709515)